api/lib/application/users/index.js

```diff
@@ -135,9 +135,10 @@ exports.register = async function(server) {
           payload: {
             data: {
               attributes: {
-                password: Joi.string().regex(XRegExp(passwordValidationPattern)),
+                password: Joi.string().regex(XRegExp(passwordValidationPattern)).allow(null),
```

Comment:
je suis toujours pas convaincue de ce `allow(null)` ... pourquoi lors de l'ajout de `pix-orga-terms-of-service-accepted` et `pix-certif-terms-of-service-accepted` cela n'a pas posé problème ?

Reply:
Parce que dans pix-orga et pix-certif le `user` n'a pas de password dans son modèle.